modules/theme/src/themes/default/views/card.variables

```diff
@@ -89,9 +89,9 @@ Recent Application Card
 @templateCardTransition: @defaultCardTransition;
 @templateCardBorder: @defaultCardBorder;
 @templateCardTextContainerPadding: 0.8em 1em;
-@templateCardHeaderFontSize: 1em;
-@templateCardHeaderFontWeight: 500;
-@templateCardHeaderLineHeight: 1.3em;
+@templateCardHeaderFontSize: 1em !important;
```

Comment:
Why do we need to add `!important` here? If it is really necessary to force, shall we do it at the definitions (.less/.override) level?

Reply:
These cards had been wrapped with a group during a recent code change[1]. Tried wrapping the classes with `.ui.cards` and it worked. So `!important` was removed with commit #3c7aa1a.

[1] https://github.com/wso2/identity-apps/blob/master/modules/react-components/src/components/grid/template-grid.tsx#L413